### ISSUE TITLE
lanranagi: 

### ISSUE THREAD

**Issue by heraplem** [NixOS/nixpkgs#337669]
### Steps To Reproduce
1. ...
2. ...
3. ...

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by SigmaSquadron** [comment]
Don't forget to notify the maintainers of the package.

---

@TomaSajt

**Comment by eclairevoyant** [comment]
Also please share the actual error messages you got

**Comment by TomaSajt** [comment]
Indeed, this looks like a regression that happened between `0.9.0` and `0.9.10`. (`0.9.21` still has the same issue)

I'm not getting the error messages, however I also noticed that the thumbnails weren't generating.

As mentioned above, I'd appreciate any error logs you can provide.

**Comment by TomaSajt** [comment]
@heraplem if you're able to, could you provide some logs?
